### DISCUSSION TITLE
cesium加载本地瓦片出现的错误

### DISCUSSION THREAD

**Discussion by kfcvs** [zouyaoji/vue-cesium#189] [Q&A] [answered]
加载本地天地图瓦片数据，0级和1级出现错乱，2级后恢复正常，如下图所示，链接代码为 const url = ref('tainditu/{z}/{y}/{reverseX}.png')，请各位大神解答

0级
![image](https://user-images.githubusercontent.com/31976998/162684699-71927ba3-d760-4651-acfc-9efff4def072.png)
1级
![image](https://user-images.githubusercontent.com/31976998/162684770-890261ae-b74b-4789-ac6a-773bbbbc92f8.png)
2级
![image](https://user-images.githubusercontent.com/31976998/162684807-4717e8e7-0640-45a2-ab91-9be47a689818.png)

**Comment by zouyaoji** [comment, marked as answer]
是用的 vc-imagery-provider-tianditu 组件吗？如果是的话 考虑瓦片是不是不太对。

哦，这个组件 还不能自定义 url。
